wls-wahlvorbereitung-service/src/main/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/service/urnenwahlvorbereitung/UrnenwahlvorbereitungService.java

```diff
@@ -35,4 +38,22 @@ public Optional<UrnenwahlvorbereitungModel> getUrnenwahlvorbereitung(@P("wahlbez
 
         return dataFromRepo.map(urnenwahlvorbereitungModelMapper::toModel);
     }
+
+    @PreAuthorize(
+        "hasAuthority('Wahlvorbereitung_BUSINESSACTION_PostUrnenwahlVorbereitung')"
+                + "and @bezirkIdPermisionEvaluator.tokenUserBezirkIdMatches(#vorbereitungToSet.wahlbezirkID, authentication)"
```

Comment:
I don't recall if spring got any better with this, but having all this logics as a plain string could be a bug/risk in the future. I would prefer to have statics defined somewhere for these checks.

Reply:
possible substring: `bezirkIdPermisionEvaluator.tokenUserBezirkIdMatches`

Reply:
it is maintainable in the IDE